src/data_provider/src/packages/packageLinuxRpmParserHelper.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Wazuh SYSINFO
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * January 28, 2021.
```

Comment:
Hi Team

I think the date is wrong

Regards
Hanes

Reply:
These are files that were copied and renamed from already existing files with this date. I'm not sure whether we are supposed to change them.